build.rs

```diff
@@ -45,7 +47,7 @@ const RING_SRCS: &[(&[&str], &str)] = &[
     (&[], "crypto/mem.c"),
     (&[], "crypto/poly1305/poly1305.c"),
 
-    (&[AARCH64, ARM, X86_64, X86], "crypto/crypto.c"),
+    (&[AARCH64, ARM, X86_64, X86, RISCV64, RISCV32], "crypto/crypto.c"),
```

Comment:
Why do we need to build crypto.c for this arch? I expect we don't need to, since crypto.c is just for architectures for which we have arch-specific non-Rust code.

Reply:
Indeed! Fixed in 71dc9e9